README.md

```diff
@@ -26,18 +26,18 @@ Directories within this repository. For the experiments in our paper, as well as
 | --------- | ----------- |
 | [docs](docs) | Documentation files (config optimizations, cov analysis, crash analysis). |
 | [examples](examples) | Target firmware samples to test Fuzzware on. |
-| [emulator](emulator) | The emulator which performs runs of a firmware for a given input file. |
+| [emulator](https://github.com/fuzzware-fuzzer/fuzzware-emulator) | The emulator which performs runs of a firmware for a given input file. |
```

Comment:
It would also be possible to mark external repositories/submodules by for example naming them `fuzzware-emulator` instead of `emulator`. So this link would got from `[emulator](https://github.com/fuzzware-fuzzer/fuzzware-emulator)` -> `[fuzzware-emulator](https://github.com/fuzzware-fuzzer/fuzzware-emulator)`

What do you think about that suggestions?